documentation-site/pages/blog/introducing-react-view/index.mdx

```diff
@@ -0,0 +1,278 @@
+        description: 'Indicates that the button is disabled',
+      },
+      overrides: {
+        value: undefined,
+        type: PropTypes.Custom,
+        description: 'Lets you customize all aspects of the component.',
+        custom: {
+          names: [],
+          sharedProps: {},
+        },
+      },
+    }}
+    scope={{
+      Button,
+      SIZE,
+    }}
+    imports={{
+      'baseui/button': {
+        named: ['Button'],
+      },
+    }}
+  />
+</div>
+
+All these parts work in harmony. An update to one of them synchronizes the state across all of them. So you can change a knob to update the source code or edit the source code to update knobs.
+
+**React View is open sourced as [uber/react-view](https://github.com/uber/react-view)** and you can get it from npm right now:
+
+```
+yarn add react-view
+```
+
+## Wait But Why?
+
+Our team is responsible for this website and the entire Base Web component library. Naturally, you might assume that most of our jobs consists of building React components. However, as the number of Base Web users grows, some days it feels more like this:
+
+<BlogImage
+  src="https://user-images.githubusercontent.com/1387913/69685208-785d8d80-1070-11ea-82bf-81bd6b7f28ee.png"
+  alt="UI Platform team's workload"
+  caption="UI Platform team's workload"
+  style={{maxWidth: '400px'}}
+/>
+
+**And that's a good thing**. If you don’t spend a lot of time helping other developers, it means that nobody uses your tools. We went from 0 to almost 500 applications at Uber in a single year! That translates into about 1000 of developers. There are 50 components, hundreds of props and endless possibilities. Is there a way to reduce the burden of support?
+
+We believe that by building na amazing documentation we can significantly reduce the number of questions in our chats and make developers happier. We have invested a significant amount of time to experiment with different kinds of documentation and React View is our latest iteration.
+
+> Build documentation so good they can’t ignore it.
+
+Does it work? Check for yourself! You can browse through our components on the left-hand side and explore various features.
+
+## How to Use React View
+
+**There are three main ways** of using React View:
+
+- **All-in-one playground**. Import `<View />`, give it the component configuration and drop it in your web documentation. This is ideal if you want to start as quickly as possible and don't need to customize anything.
+- **Build your own playground**. Import the `useView` hook and give it the component configuration. This hook handles the playground state and returns various props and callbacks that you can fit into your own UI components. React View also exports all default UI parts separately (`Editor`, `ActionButtons`, `Compiler`, `Knobs`, `Error`...) so you can reuse them. This is a great option if you want to customize some parts of the default UI (or all of it) and not to worry about the rest.
```

Comment:
that you can fit into your own UI components -> that you can use in your own UI components
and not to worry about the rest -> without worrying about the rest